app/views/home/index.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <div class="alert alert-info alert-dismissable" role="alert">
   <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
-  <h6 class="mb-0">A vacinação contra a gripe no período de 6 a 16 de abril de 2020 será exclusiva para <strong>pessoas acima de 60 anos</strong>. Após esse período, outros públicos serão atendidos. O agendamento poderá ser feito pela própria pessoa a ser vacinada, por um familiar ou outra pessoa próxima.</h6>
+  <h6 class="mb-0">A vacinação contra a gripe no período de 16 de abril a 9 de maio de 2020 será exclusiva para <strong>pessoas acima de 60 anos ou com doenças crônicas</strong>. Após esse período, outros públicos serão atendidos. O agendamento poderá ser feito pela própria pessoa a ser vacinada, por um familiar ou outra pessoa próxima.</h6>
 </div>
 
```

Comment:
Hello, here we could put a pdf file with the description of the chronic diseases "allowed" to register in system for guide better the citizens that want schedule your vaccine:
 
```suggestion
<p class="mt-5">Verifique <strong><%= link_to "Aqui", "/cartaz_doentes_cronicos.pdf", target: :_blank %></strong> a lista de doenças crônicas para que o cidadão possa realizar o agendamento da vacina.</p>
```

In this case, the `cartaz_doentes_cronicos.pdf` be in `public` folder.